src/log.js

```diff
@@ -9,11 +9,12 @@ const logLevels = {
 };
 
 // Central logging. console.log can be replaced by writing to a logfile for example
-const log = (level, message) => {
+export default function log(level, message) {
   if (logLevels[level] === undefined) {
     log('error', `Log level '${level}' was declared but not handled`);
     level = 'warn';
-  } else if (logLevels[level]) {
+  }
+  if (logLevels[level]) {
```

Comment:
this was a bug, if no level was given, the `else if` ensured the log was dropped, instead of moved to `warn`